docs/javascript-configuration.md

```diff
@@ -8,6 +8,41 @@ sidebar_label: Configuration
 
 The behaviour of the Scope Agent can be modified using environment variables or a configuration file `scope.yml` in the source root of the project.
 
+An example `scope.yml`:
+
+```yaml
+service: "service-name"
+repository: "https://github.com/undefinedlabs/scope-docs.git"
+commit_sha: "974c3566eb8e221d130db86a7ce1f99703fe2e69"
+branch: "master"
```

Comment:
```suggestion
```
Because the commit and branch info is highly dynamic, I wouldn't recommend setting it in the yaml file